DoSomethingArchive/legacy-website
# Yahoo Mobile CTA's

#### What's this PR do?

For a set of nodes specified in the [JSON file hosted on Gist](https://gist.github.com/sbsmith86/f62cd7644f36df0e1be5) it creates a mobile version of the CTA button on 11 facts pages that has a new title/button/link to a pre-filled yahoo search page. 

Also puts all of this in under a feature flag so we can turn the partnership on/off.
#### Where should the reviewer start?

Start here `dosomething_fact_page.module`.
#### How should this be manually tested?

Turn on the yahoo partnership in the theme settings and go to one of the nodes in the JSON file. On mobile, you should see the new CTA and on desktop you should see the regular experience.
#### Any background context you want to provide?

All for yahoo partnership.
#### What are the relevant tickets?

Addresses #5237
